Sources/SPTDataLoaderFactory.m

```diff
@@ -31,7 +31,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface SPTDataLoaderFactory () <SPTDataLoaderRequestResponseHandlerDelegate, SPTDataLoaderAuthoriserDelegate>
+@interface SPTDataLoaderFactory () <SPTDataLoaderRequestResponseHandlerDelegate, SPTDataLoaderAuthoriserDelegate, SPTDataLoaderRequestResponseHandler>
```

Comment:
Bug fix: `SPTDataLoaderFactory` is used as a `SPTDataLoaderRequestResponseHandler` but didn’t formally conform to it.